### PR TITLE
add tag `stale_command` in ASYNC_REQUESTS_COUNTER_VEC.

### PR DESCRIPTION
**Pull request by hicqu**
add tag `stale_command` in ASYNC_REQUESTS_COUNTER_VEC.

### CODE REVIEW

**Comment by hicqu** [comment]
@BusyJay @overvenus @siddontang PTAL thanks!

File: src/storage/mod.rs [hunk 1]
```diff
@@ -977,6 +977,8 @@ pub fn get_tag_from_header(header: &errorpb::Error) -> &'static str {
         "stale_epoch"
     } else if header.has_server_is_busy() {
         "server_is_busy"
+    } else if header.has_stale_command() {
+        "stale_command"
```
Comment (overvenus) [inline src/storage/mod.rs]: There are two errors not covered yet, `StoreNotMatch` and `RaftEntryTooLarge`.

**Comment by BusyJay** [comment]
Any updates?

**Comment by hicqu** [comment]
@overvenus @BusyJay PTAL again, thanks.

File: src/storage/mod.rs [hunk 2]
```diff
@@ -977,8 +977,14 @@ pub fn get_tag_from_header(header: &errorpb::Error) -> &'static str {
...
+    } else if header.has_store_not_match() {
+        "store_not_match"
+    } else if header.has_raft_entry_too_large() {
+        "raft_entry_too_large"
     } else {
-        "other"
+        unreachable!();
```
Comment (BusyJay) [inline src/storage/mod.rs]: Should not panic as we may add more error type in the future.

**Comment by hicqu** [comment]
Fixed. @overvenus @BusyJay PTAL, thanks.

**Pull request merged**